index/index.go

```diff
@@ -868,9 +868,9 @@ func (r *Reader) LabelValues(names ...string) (StringTuples, error) {
 		return nil, errors.Wrap(d.err(), "read label value index")
 	}
 	st := &serializedStringTuples{
-		l:      nc,
-		b:      d.get(),
-		lookup: r.lookupSymbol,
+		length:  nc,
+		breadth: d.get(),
```

Comment:
what does `breadth` stand for?

Reply:
I was confused as [here](https://github.com/prometheus/tsdb/pull/377/files#diff-71ebe2bcf31a915b1fa3b3b289d5d31dR977) in the definition of struct **serializedStringTuples** it was previously slice of bytes and didn't knew what it should be. What do you suggest should I change it to?

Reply:
@krasi-georgiev after reading code deeper and some [go doc](https://golang.org/pkg/encoding/binary/#example_ByteOrder_get) I found that its not `breadth` its just **slices of binary data**. So instead of binary I changes it to *binSlice*. And *l* can be used as length because it is just size of serializedString tuple. I am saying beacause [here](https://github.com/prometheus/tsdb/blob/master/index/index.go#L864) **nc** is being initialized as `be32int` that is an integer returned from [here](https://github.com/prometheus/tsdb/blob/master/index/encoding_helpers.go#L78). And also we are using **l** just to iterate [here](https://github.com/prometheus/tsdb/blob/master/index/index.go#L991). So as per me *lenght* is just fine. If you suggest anything else I can change it to that. I have pushed latest changes please review.

Reply:
@krasi-georgiev updated.